http-builder-ng-core/src/main/java/groovyx/net/http/FromServer.java

```diff
@@ -404,7 +422,8 @@ public Long parse() {
             tmp.put("Refresh", CombinedMap::new);
             tmp.put("Retry-After", HttpDate::new);
             tmp.put("Server", ValueOnly::new);
-            tmp.put("Set-Cookie", MapPairs::new);
+            tmp.put("Set-Cookie", HttpCookies::new);
+            tmp.put("Set-Cookie2", HttpCookies::new);
```

Comment:
Set-Cookie2 is actually the signal that the server is expecting version 1 of cookies, not the version 0 netscape spec for cookies.